server/views/formPages/ratings/furtherCharges.html

```diff
@@ -60,7 +59,7 @@ <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
     ]
   }) }}
 
-  {% set guidancePrevious %}<div class="govuk-!-font-weight-bold govuk-!-margin-top-2">You are looking for:</div>
+  {% set guidancePrevious %}<div class="govuk-!-font-weight-bold govuk-!-margin-top-2">Category B might be appropriate if this person has:</div>
```

Comment:
Should this be: "What should I look for?" ?

Reply:
corrected